docker-compose-cluster.yml

```diff
@@ -0,0 +1,85 @@
+version: '3'
+services:
+  rabbitmq:
+    image: 'rabbitmq:3.8.2-management-alpine'
+    ports:
+      - '5672:5672'
+      - '15672:15672'
+    container_name: 'ziggurat_rabbitmq'
+
+  zookeeper:
+    image: zookeeper:3.4.9
+    hostname: zookeeper
+    ports:
+      - "2181:2181"
+    environment:
+      ZOO_MY_ID: 1
+      ZOO_PORT: 2181
+      ZOO_SERVERS: server.1=zookeeper:2888:3888
```

Comment:
Why `server.1`?

Reply:
ZOO_MY_ID is a mandatory env variable and ZOO_SERVERS is used to specify the list of zookeeper machines.
server.1 means zookeeper machine with ZOO_MY_ID = 1